include/seqan/sequence/adapt_std_string.h

```diff
@@ -163,6 +163,21 @@ struct DefaultOverflowImplicit< std::basic_string<TChar, TCharTraits, TAlloc> >
 template <typename TChar, typename TCharTraits, typename TAlloc>
 struct IsSequence<std::basic_string<TChar, TCharTraits, TAlloc> > : True {};
 
+// ----------------------------------------------------------------------------
+// Metafunction Chunk
+// ----------------------------------------------------------------------------
+
+// Chunk interface for std::basic strings.
+template <typename TChar, typename TCharTraits, typename TAlloc, typename TSpec>
+struct Chunk<Iter<std::basic_string<TChar, TCharTraits, TAlloc>, AdaptorIterator<TChar*, TSpec> > >
+{
+    typedef typename Chunk<std::basic_string<TChar, TCharTraits, TAlloc> >::Type Type;
+};
```

Comment:
Mmh. I think we can do chunking for std::vector and std::string in the same way without using private members. The only problem I see is that before C++11 you couldn't rely to get contiguous memory chunks from the string, i.e. you cannot simply convert the iterators into a char* (In practice all standard STL implementations use contiguous memory, but at least the TBB not). The second problem might be that it would be not compliant to write in reserved/not-resized storage. 

Hence reserveChunk must use resize and I have to think about how to implement advanceChunk then.